lib/typescript/httpclient/mappers/connectChannelApiMapper.ts

```diff
@@ -0,0 +1,11 @@
+import {ConnectChannelRequestPayload, ConnectChannelRequestApiPayload} from '../payload';
+
+export const connectChannelApiMapper = (payload: ConnectChannelRequestPayload): ConnectChannelRequestApiPayload => {
```

Comment:
nitpick: For the mappers, you can simplify a bit by writing
```
export const fn = (payload) => ({
your mapping code
})
```
instead of instantiating objects first and calling return

Reply:
thanks for the refactoring tip! fixing now!